src/libraries/Common/tests/LongName/LongName.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>LongName</AssemblyName>
```

Comment:
This doesn't seem to be a very long name. 
Seems that the length requirement is in the full path and not in the assembly name, right? If so, do we really want to name this assembly `LongName`?

Reply:
> Seems that the length requirement is in the full path and not in the assembly name, right?

You are right.

> If so, do we really want to name this assembly LongName?

Would `LongPath` be a better name?

Reply:
whatever you decide on, please drop the assemblyname property then as it defaults to the project's name.

Reply:
>Would LongPath be a better name?

I would prefer a name unrelated to the test so this could be reused in the future (e.g: TestProject), I don't have an strong opinion though.